index.html

```diff
@@ -0,0 +1,393 @@
+        </div>
+        <div id="speaker3" class="speaker">
+            <div class="speaker-image">
+                <img src="./assets/img/speaker3.png" alt="">
+            </div>
+            <div class="speaker-info">
+                <div class="speaker-name">
+                    <a href="#">ZikaZaki</a>
+                </div>
+                <div class="org-name">MIT Institute</div>
+                <div class="speaker-desc">
+                    Focusing on a collaborative approach in a networked environment, 
+                    we created the concept of co-production based on sharing, 
+                    such as open source software and Wikipedia.
+                </div>
+            </div>
+        </div>
+        <div id="speaker4" class="speaker">
+            <div class="speaker-image">
+                <img src="./assets/img/speaker4.png" alt="">
+            </div>
+            <div class="speaker-info">
+                <div class="speaker-name">
+                    <a href="#">ZikaZaki</a>
+                </div>
+                <div class="org-name">MIT Institute</div>
+                <div class="speaker-desc">
+                    Focusing on a collaborative approach in a networked environment, 
+                    we created the concept of co-production based on sharing, 
+                    such as open source software and Wikipedia.
+                </div>
+            </div>
+        </div>
+        <div id="speaker5" class="speaker">
+            <div class="speaker-image">
+                <img src="./assets/img/speaker5.png" alt="">
+            </div>
+            <div class="speaker-info">
+                <div class="speaker-name">
+                    <a href="#">ZikaZaki</a>
+                </div>
+                <div class="org-name">MIT Institute</div>
+                <div class="speaker-desc">
+                    Focusing on a collaborative approach in a networked environment, 
+                    we created the concept of co-production based on sharing, 
+                    such as open source software and Wikipedia.
+                </div>
+            </div>
+        </div>
+        <div id="speaker6" class="speaker">
+            <div class="speaker-image">
+                <img src="./assets/img/speaker6.png" alt="">
+            </div>
+            <div class="speaker-info">
+                <div class="speaker-name">
+                    <a href="#">ZikaZaki</a>
+                </div>
+                <div class="org-name">MIT Institute</div>
+                <div class="speaker-desc">
+                    Focusing on a collaborative approach in a networked environment, 
+                    we created the concept of co-production based on sharing, 
+                    such as open source software and Wikipedia.
+                </div>
+            </div>
+        </div>
+    </div>
+   </section>
+    <!-- ===============End of The Speakers Section================ -->
```

Comment:
- [ ] The speakers section should be generated with the javascript. You shouldn't use HTML to create this section.